docs/container/README.md

```diff
@@ -77,6 +77,33 @@ Once the FalconContainer resource is pushed to the cluster the operator will sta
 | conditions.["InstallerComplete"]    | Informs about completion of Falcon Container Installer. Users can review the installer Job/Pod in `falcon-system-configure` namespace     |
 | conditions.["Complete"]             | Informs about the completion of the deployment of Falcon Container                                                                        |
 
+### Image Registry considerations
+
+Falcon Container Image is distributed by CrowdStrike through CrowdStrike Falcon registry. Operator supports two modes of deployment:
+
+#### (Option 1) Use CrowdStrike registry directly
+
+This is recommended for demoing purposes. Does not require any advanced set-ups. Users are advised to use the following excerpt in theirs FalconContainer custom resource definition.
```

Comment:
Why is this only recommended for demoing and production?

Reply:
I don't think I meant to say `only`, but not being native speaker makes sometimes these little details challenging. Does the above excerpt imply `only` to You?

Reply:
right... sometimes I forget that...

Personally, I would remove the sentence altogether since it more reads as "not recommended" for any thing other than demos.

Reply:
Got it. Thanks!